ivy/stateful/initializers.py

```diff
@@ -75,7 +85,13 @@ def create_variables(self, var_shape, device, fan_out=None, fan_in=None):
                 "fan_sum | fan_avg ] "
             )
         wlim = ((self._numerator / fan) ** self._power) * self._gain
-        return ivy.variable(ivy.random_uniform(-wlim, wlim, var_shape, device=device))
+        return ivy.variable(
+            ivy.array(
```

Comment:
`ivy.array` is redundant here

Reply:
Fixed.

Reply:
good point, actually we should change `ivy.random_uniform` so that it does support the `dtype` argument.